www/templates/default/html/Controller.tpl.php

```diff
@@ -39,6 +40,7 @@
 $page->addStyleDeclaration("#dcf-mobile-toggle-menu {display: none!important}");
 
 //javascript
+$page->addScriptDeclaration('WDN.setPluginParam("idm", "logout", "' . Util::getBaseURL() . '/manager/logout");');
```

Comment:
Not generating the correct logout link. I'm getting "https://events-dev.unl.edu/TractorSafetyTraining/https/manager/logout"